### PR TITLE
chore: documentation for record

### PR DESCRIPTION
**Pull request by alexanderblv**
## Motivation
Closes https://github.com/AleoNet/snarkVM/issues/2132
(Write your motivation here)

## Test Plan

(Write your test plan here)

## Related PRs

(Link your related PRs here)

### CODE REVIEW

**Comment by vicsn** [comment]
Thx for the chore! Do you mind keeping the original comment (adding your new line above it)? And doing the same for the function below it? I assume the original line was useful for the original author and may be for future readers.

**Comment by alexanderblv** [comment]
> Thx for the chore! Do you mind keeping the original comment (adding your new line above it)? And doing the same for the function below it? I assume the original line was useful for the original author and may be for future readers.

Done!

**Review by vicsn — APPROVED** [review]

**Comment by alexanderblv** [comment]
@vicsn It seems I don't have the rights to put this PR to merge queue. Could you do that?

**Comment by vicsn** [comment]
I don't either, it's not high priority so can be a while

File: console/program/src/data/record/is_owner.rs [hunk 1]
```diff
@@ -15,6 +15,7 @@
 use super::*;
 
 impl<N: Network> Record<N, Ciphertext<N>> {
+    /// Determines whether the record belongs to the account.
```
Comment (raychu86) [inline console/program/src/data/record/is_owner.rs:18]:
```suggestion
    /// Returns `true` if the given view key is the owner of the record. 
```

**Review by raychu86 — CHANGES_REQUESTED** [review]

**Comment by raychu86** [comment]
Closing in favor of https://github.com/AleoNet/snarkVM/pull/2513

**Pull request closed without merging**